docs/resources/aws_emr_clusters.md

```diff
@@ -0,0 +1,68 @@
+---
+title: About the aws_emr_clusters Resource
+platform: aws
+---
+
+# aws\_emr\_clusters
+
+Use the `aws_emr_clusters` resource to test the configuration of a collection of AWS Elastic MapReduce Service.
+
+## Syntax
+```ruby
+describe aws_emr_clusters do
+  its('CLUSTER_IDS') { should include 'GIVEN_CLUSTER_ID'}
+end
+```
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on EMR Clusters](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html).
+
+## Properties
+
+|Property                                | Description|
```

Comment:
In the Plural resource, there are parameters you have not added. Parameters that are not added are code, message, creation_date)time, ready_date_time, end_date_time, normalized_instance_hours. Can you please add these attributes?

Reply:
Please add the fields column also. Please refer to other PRs that we are following.